Packs/TrendMicroCAS/Integrations/TrendMicroCAS/TrendMicroCAS.yml

```diff
@@ -38,6 +44,8 @@ configuration:
   - exchangeserver
   required: false
   type: 16
+  section: Collect
+  advanced: true
```

Comment:
```suggestion
  section: Collect
```
I think it will not fetch anything if this is empty

Reply:
Its not required and the code seems to handle empty case..